commercetools.Sdk/commercetools.Sdk.Client/GetMatchingQueryCommand.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using commercetools.Sdk.Domain;
+using commercetools.Sdk.Domain.Query;
+
+namespace commercetools.Sdk.Client
+{
+    public abstract class GetMatchingQueryCommand<T> : Command<PagedQueryResult<T>>, IExpandable
+    {
+        protected GetMatchingQueryCommand()
+        {
+            this.Expand = new List<string>();
+            this.UrlSuffix = string.Empty;
+        }
+
+        protected GetMatchingQueryCommand(IAdditionalParameters<T> additionalParameters)
+        {
+            this.AdditionalParameters = additionalParameters;
+        }
+
+        public override System.Type ResourceType => typeof(T);
+
+        public List<string> Expand { get; set; }
+
+        public string UrlSuffix { get; set; }
```

Comment:
Maybe remove the set?

`public virtual string UrlSuffix => string.Empty`

Reply:
:thumbsup: